### PR TITLE
Add ohakola chall01

### PR DESCRIPTION
**Pull request by hakolao**
Done!

### CODE REVIEW

File: chall01/ohakola.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+import sys
+
+usage = "usage: ./ohakola.py <a-zA-Z string>"
```
Comment (jgengo) [inline chall01/ohakola.py:4]: nice! but don't hardcode your app name when you shouldn't use `sys.argv[0]`

File: chall01/ohakola.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+			char = str[i]
+			if (i < str_len - 1):
+				print(morse_dict[char], end = '')
+			else:
+				print(morse_dict[char], end = '\n')
+			i += 1
+	else:
+		print(usage)
+
+print_args()
```
Comment (jgengo) [inline chall01/ohakola.py:53]: A good practice is to protect your python scripts that are meant to be run from running on import.
It's done by
```python
if __name__ == "__main__":
  print_args()
```
and then your code (or even better would be to put your code to function called main() and call it here)

File: chall01/ohakola.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	"w": ".--",
+	"x": "-..-",
+	"y": "-.--",
+	"z": "--.."
+}
+
+def print_args():
+	args_len = len(sys.argv)
+	str = sys.argv[1].lower()
+	if (args_len == 2):
```
Comment (jgengo) [inline chall01/ohakola.py:37]: I would recommend checking your args length outside of this function

File: chall01/ohakola.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+		str_len = len(str)
+		if (str_len == 0):
+			print(usage)
+			return
```
Comment (jgengo) [inline chall01/ohakola.py:41]:
```python
if not str:
```

File: chall01/ohakola.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+		i = 0
+		while (i < str_len):
+			char = str[i]
+			if (i < str_len - 1):
+				print(morse_dict[char], end = '')
```
Comment (jgengo) [inline chall01/ohakola.py:46]: You should append into a string and then print at the end to avoid using print for every character.

**Review by jgengo — APPROVED** [review]

File: chall01/ohakola.py (hunk 2 above)
Comment (hakolao) [reply chall01/ohakola.py:53]: Thanks so much for comments! Good points all :)

**Pull request merged**